src/OrbitGl/Batcher.cpp

```diff
@@ -265,19 +276,23 @@ void Batcher::AddTriangle(const Triangle& triangle, const std::array<Color, 3>&
   user_data_.push_back(std::move(user_data));
 }
 
-void Batcher::AddCircle(Vec2 position, float radius, float z, Color color) {
+Vec3 Batcher::TransformVertex(const Vec3 input) const {
+  return Vec3(floorf(input[0]), floorf(input[1]), input[2]) + current_translation_;
```

Comment:
Shouldn't we floorf() after having applied the translation?

Reply:
Yes, thanks. I've changed that.